### PR TITLE
[KED-3046] Remove "QuantumBlack Labs" from places where it doesn't belong any more

### PR DESCRIPTION
**Pull request by SajidAlamQB**
## Description

There's a few mentions that kedro is authored by QuantumBlack Labs (e.g. in kedro docs, setup.,py, `kedro info`). Following the migration to LF these shouldn't be there any more.

Best way to find them is just to search "quantumblack" in the codebase - there's not that many results to check through.

Additionally you will also have to update URLs for Kedro-Airflow, Kedro-Docker and Kedro-Telemetry as they have completely changed repo location.

As well as kedro repository, you should check kedro-community, kedro-plugins, kedro-viz and kedro-training. kedro-starters has already been updated.

https://jira.quantumblack.com/browse/KED-3046

## Checklist

- [x] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes

### CODE REVIEW

**Review by lorenabalan — APPROVED** [review]
Excellent. 👌

**Comment by AntonyMilneQB** [comment]
Looks good so far, but I think there's a few more places which might need updating:
* recently merged `docs/source/03_tutorial/07_set_up_experiment_tracking.md` points to the old kedro-viz repo URL
* do we still want to use kedro@quantumblack.com - @yetudada?
* very minor, but the stuff under "Set git email and name" in CircleCI config

File: docs/conf.py [hunk 1]
```diff
@@ -280,7 +280,7 @@
         "Kedro Documentation",
         author,
         "Kedro",
-        "Kedro is a Data Science framework for QuantumBlack-led projects.",
+        "Kedro is an open source Data Science framework.",
```
Comment (yetudada) [inline docs/conf.py:283]:
```suggestion
        "Kedro is a Python framework for creating reproducible, maintainable and modular data science code.",
```

File: kedro/framework/cli/cli.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ def info():
     click.echo(
         "kedro allows teams to create analytics\n"
         "projects. It is developed as part of\n"
-        "the Kedro initiative at QuantumBlack."
+        "the Kedro initiative."
```
Comment (yetudada) [inline kedro/framework/cli/cli.py:66]: Might I suggest this text becomes: 
"Kedro is a Python framework for\n"
"creating reproducible, maintainable\n"
"and modular data science code."

File: docs/source/09_development/03_commands_reference.md [hunk 3]
```diff
@@ -119,7 +119,7 @@ v0.17.6
 
 kedro allows teams to create analytics
 projects. It is developed as part of
-the Kedro initiative at QuantumBlack.
+the Kedro initiative.
```
Comment (yetudada) [inline docs/source/09_development/03_commands_reference.md:122]: Might I suggest this text becomes: 
"Kedro is a Python framework for\n"
"creating reproducible, maintainable\n"
"and modular data science code."

**Review by yetudada — COMMENTED** [review]
Approved and I think let's remove the `kedro@quantumblack.com` email addresses from the code base and instead tell users to raise GitHub issues or reach us on Discord to find us.

**Review by MerelTheisenQB — APPROVED** [review]
👍

**Pull request merged**